program_intent_engagement/apps/api/serializers.py

```diff
@@ -1,4 +1,33 @@
-# Serializers that can be shared across multiple versions of the API
-# should be created here. As the API evolves, serializers may become more
-# specific to a particular version of the API. In this case, the serializers
-# in question should be moved to versioned sub-package.
+"""
+Serializers for the program-intent-engagement API
+"""
+from rest_framework import serializers
+
+from program_intent_engagement.apps.core.models import ProgramIntent, CERTAINTY
+
+class ProgramIntentSerializer(serializers.ModelSerializer):
+    """
+    Serializer for the Program Intent Model
+    """
+
+    # TODO: gotta figure out the JWT stuff for this
```

Comment:
that's not really relevant in the serializer (which has a user) so if this TODO is still to be done it goes in the endpoint itself

Reply:
oops forgot to remove this comment